portal/templates/profile_macros.html

```diff
@@ -836,9 +836,11 @@ <h4 class="text-muted">{{ _("Questions inquired of the patient at registration")
         };
         $(function () {
             $("#patientQ").show();
+            $("#patTx").remove();
```

Comment:
Paul, I can't remove it from the source (which is coming from initia_queries_macros.html).  If I do that, the question won't display on the initial queries page (i.e. but we still need the question there).  So to work around that, I basically just remove the question from the rendering DOM on the profile page, therefore effectively making treatment question unavailable only on this page.